scalnet/pom.xml

```diff
@@ -228,7 +164,7 @@
                     <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
                     <junitxml>.</junitxml>
                     <filereports>WDF TestSuite.txt</filereports>
-                    <skipTests>${maven.test.skip}</skipTests>
+                    <skipTests>${scala.test.skip}</skipTests>
```

Comment:
Let's keep this as maven.test.skip - otherwise won't we run scalnet tests even if `maven.test.skip=true` is passed on command line?

Reply:
`scala.test.skip` should be fine, because we have updated root pom config for skipping tests:
https://github.com/KonduitAI/deeplearning4j/pull/480/files#diff-600376dffeb79835ede4a0b285078036R608-R626